app/assets/javascripts/spree/frontend/add_recently_viewed_products.js.coffee

```diff
@@ -0,0 +1,5 @@
+Spree.fetch_recently_viewed_products = (productId = "")->
+  $.ajax
```

Comment:
@priyank-gupta we want to fetch this also on products list, taxon pages and so on, prouctId is needed only for adding a product to recently viewed list